open-metadata-resources/open-metadata-deployment/charts/vdc/templates/ibm-igc.yaml

```diff
@@ -174,7 +174,7 @@ spec:
                 name: {{ include "myapp.fullname" . }}-configmap
           env:
             - name: "logging.level.org.odpi.openmetadata.adapters.repositoryservices.igc"
-              value: "DEBUG"
+              value: {{ .Values.logging.level }}
```

Comment:
We should probably just drop this environment variable from the default config (or inherit the top-level logging rather than this IGC-specific-level) (?)

Reply:
Currently that fragment does inherit a common logging value from the template -- obviously if the user of that variable within the code changes then we should update accordingly.

The value is defaulted to no logging on the principle that debug should be possible via audit logs.

Reply:
suggest we push add-is, and if a change is needed go with new issue.
I think the proposed change is a step forward.